tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_beacon.c

```diff
@@ -244,6 +372,9 @@ static const struct bst_test_instance test_beacon[] = {
 
 	TEST_CASE(rx, on_iv_update,   "Beacon: receive with IV update flag"),
 	TEST_CASE(rx, on_key_refresh,  "Beacon: receive with key refresh flag"),
+
+	TEST_CASE(tx_beacon, secure, "Beacon: send secure mesh beacons"),
```

Comment:
This sounds a bit generic. I would rephrase the test name to emphasize that its purpose is to check beacon interval.